src/main/java/competition/subsystems/armlifting/LiftingSubsystem.java

```diff
@@ -0,0 +1,53 @@
+package competition.subsystems.armlifting;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+ 
+@Singleton
+public class LiftingSubsystem extends BaseSubsystem {
+    
+    final DoubleProperty liftingPowerProp;
+    final DoubleProperty loweringPowerProp;
+    int maximumLiftHeight;
+    int currentLiftHeight;
+
+
+    @Inject
+    public LiftingSubsystem(CommonLibFactory factory, PropertyFactory pf) {
+        log.info("Creating LiftingSubsystem");
+        pf.setPrefix(this);
+        liftingPowerProp = pf.createPersistentProperty("Lifting Power", 1);
+        loweringPowerProp = pf.createPersistentProperty("Lowering Power", 1);
+
```

Comment:
In general we can remove trailing blank lines like this one

Reply:
done